src/bindings/c/include/openvino/c/ov_model.h

```diff
@@ -58,18 +58,84 @@ OPENVINO_C_API(ov_status_e) ov_model_inputs(const ov_model_t* model, ov_output_n
  * @return Status code of the operation: OK(0) for success.
  */
 OPENVINO_C_API(ov_status_e)
-ov_model_input_by_name(const ov_model_t* model, const char* tensor_name, ov_output_const_node_t** input_node);
+ov_model_const_input_by_name(const ov_model_t* model, const char* tensor_name, ov_output_const_node_t** input_node);
 
 /**
  * @brief Get the outputs of ov_model_t.
```

Comment:
Comment is incorrect

Reply:
Get port object? @ilyachur

Reply:
This is documentation for method which works with input...
I am not against such words as inputs/outputs.